.github/workflows/CI-python-vision.yml

```diff
@@ -0,0 +1,146 @@
+name: CI Python Vision
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+    paths:
+      - "erroranalysis/**"
+      - "rai_test_utils/**"
+      - ".github/workflows/CI-python-vision.yml"
+
+jobs:
+  ci-python:
+    strategy:
+      matrix:
+        packageDirectory:
+          ["erroranalysis", "rai_test_utils"]
+        operatingSystem: [ubuntu-latest, macos-latest, windows-latest]
+        pythonVersion: ["3.8", "3.9", "3.10"]
+        exclude:
+          - operatingSystem: macos-latest
+            pythonVersion: "3.8"
+
+    runs-on: ${{ matrix.operatingSystem }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.pythonVersion }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.pythonVersion }}
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          python-version: ${{ matrix.pythonVersion }}
+
+      - if: ${{ matrix.operatingSystem != 'macos-latest' }}
+        name: Install pytorch on non-MacOS
+        shell: bash -l {0}
+        run: |
+          conda install --yes --quiet pytorch torchvision captum cpuonly -c pytorch
+
+      - if: ${{ matrix.operatingSystem == 'macos-latest' }}
+        name: Install Anaconda packages on MacOS, which should not include cpuonly according to official docs
+        shell: bash -l {0}
+        run: |
+          conda install --yes --quiet pytorch torchvision captum -c pytorch
+
+      - if: ${{ matrix.operatingSystem == 'macos-latest' }}
+        name: Install latest lightgbm from conda-forge for MacOS
+        shell: bash -l {0}
+        run: |
+          conda install --yes --quiet lightgbm -c conda-forge
+
+      - name: Setup tools
+        shell: bash -l {0}
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade setuptools
+          pip install --upgrade "pip-tools<=7.1.0"
+
+      - name: Install package dependencies
+        shell: bash -l {0}
+        run: |
+          pip install -r requirements-dev.txt
+        working-directory: ${{ matrix.packageDirectory }}
+
+      - name: Install package extras
+        shell: bash -l {0}
+        run: |
+          pip install -r requirements-object-detection.txt
+        working-directory: ${{ matrix.packageDirectory }}
+
+      - name: Install package
+        shell: bash -l {0}
+        run: |
+          pip install -v -e .
```

Comment:
So this is without the extras (which would be pip install .[object-detection]). What about with extras? I'm a bit confused honestly, because  you installed the requirements for the extras above. So this is neither really (?)

Reply:
@romanlutz I installed `requirements-object-detection.txt` under the Install extras name. `pip install .[object_detection]` would just install this file under the hood; let me know if I should replace directly installing from test file to using the extras term?